Tests/XpringClientIntegrationTests.swift

```diff
@@ -1,6 +1,7 @@
 import XCTest
 import XpringKit
 
+/*
```

Comment:
extensions occur globally so these two things are going to trample on each other. If this wasn't test code we should probably name these things more generically and / or have them in their own files. Since this test file will go away soon, I think it's fine to just rely on them being added in `XRPClientIntegrationTests`.  Thus, you can delete this block comment.